bucket/yt-dlp.json

```diff
@@ -24,7 +24,7 @@
                 "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp.exe"
             },
             "32bit": {
-                "url": "https://github.com/yt-dlp/yt-dlp/releases/download/$version/yt-dlp_x86.exe#/yt-dlp.exe"
```

Comment:
Why did you change this?

Reply:
@rashil2000 It seems they changed the URL in the last version. I replaced with the correct one.

Reply:
No, you need to keep the `#/yt-dlp.exe`, it's not a part of the URL, it is for Scoop to rename the file.